src/types/actionTypes.ts

```diff
@@ -0,0 +1,24 @@
+import { Todo, TodosListType } from './todosTypes';
+
+export type Actions = {
+  type: 'LOAD',
+  payload: TodosListType,
+} | {
+  type: 'POST',
+  payload: Todo,
+} | {
+  type: 'DELETE',
+  payload: number,
+} | {
+  type: 'PATCH',
+  payload: Todo,
+} | {
+  type: 'ALL_ACTIVE',
+  payload: null,
+} | {
+  type: 'IS_SPINNING',
+  payload: number,
+} | {
+  type: 'REMOVE_SPINNING',
+  payload: number,
+};
```

Comment:
It's not readable, create a type for each action and then combine them with union type